boards/components/src/sht3x.rs

```diff
@@ -0,0 +1,100 @@
+//! Component for the SHT3x sensor.
+//!
+//! I2C Interface
+//!
+//! Usage
+//! -----
+//!
+//! With the default i2c address
+//! ```rust
+//! let sht3x = components::sht3x::SHT3xComponent::new(sensors_i2c_bus, mux_alarm).finalize(
+//!         components::sht3x_component_helper!(nrf52::rtc::Rtc<'static>),
+//!     );
+//! sht3x.reset();
+//! ```
+//!
+//! With a specified i2c address
+//! ```rust
+//! let sht3x = components::sht3x::SHT3xComponent::new(sensors_i2c_bus, mux_alarm).finalize(
+//!         components::sht3x_component_helper!(nrf52::rtc::Rtc<'static>, capsules::sht3x::BASE_ADDR << 1),
```

Comment:
is the address shift in this example still needed?

Reply:
Yes, in the main is used in the same way, because i2c address must be fixed to use the proper address.

Reply:
Are you sure that still applies after #2272?

Reply:
I didn't see #2272, now I rebased and there shouldn't be any conflicts.

Reply:
It seems you've adjusted the main accordingly (to not require a shift), although the comment is still out of date?

Reply:
Sorry, I forgot about the comment. I solved it now.